djtriggers/tests/test_logic.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta
+
+import mock
+from django.test import override_settings
+from django.test.testcases import TestCase
+from django.utils import timezone
+
+from djtriggers.logic import process_triggers
+from djtriggers.tests.factories.triggers import NormalDummyTriggerFactory
+
+
+class LogicTest(TestCase):
+
+    def populate_db(self, quantity, factory, **kwargs):
+        return [factory(**kwargs) for _ in xrange(quantity)]
```

Comment:
I like this approach :) but for clarity i'd make the quantity a named field (probably personal preference though)